scripts/uniffi_bindgen_generate_go.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Should we add a comment that cross is needed?

https://github.com/cross-rs/cross

What arguments do I need to pass to run this on Linux?